src/extension/iSamples_rangeSlider.js

```diff
@@ -0,0 +1,172 @@
+import PropTypes from 'prop-types';
```

Comment:
Again, this is identical to the one in `solr-faceted-search-react`.  I'll investigate if we can just export it.

Reply:
This is a js library. I think we already installed it in our app.